alerts/src/commonMain/kotlin/Alerts.kt

```diff
@@ -119,6 +133,7 @@ abstract class BaseAlertPresenter(private val alert: Alert) : AlertActions {
         private var title: String? = null
         private var message: String? = null
         private var actions: MutableList<Alert.Action> = mutableListOf()
+        private var textInputAction: Alert.TextInputAction? = null
```

Comment:
I think it should be a list. Image input of login and password same time

Reply:
I actually have not thought of creating a multi input dialog. I will investigate and come back to you.

Reply:
Unfortunately because of the way the call back is declared for the UITextField it can not be done or at least I could not figure out how. 

You can check the limitation of having to declare :

```
@ObjCAction
fun textDidChange() {..}

```    
If you have an idea for a solution I could implement it. For now I will close this remark.

Reply:
Your `testDidChange` should accept UITextField as parameter, so you can know which one you have text for